lib/fs.js

```diff
@@ -62,7 +64,72 @@ if (version[0] < 10 || (version[0] === 10 && version[1] <= 11)) {
   };
 }
 
+// Recursively remove directory
+//   file <string> file or directory to be removed
+//   cb <Function> callback
+const rmRecursive = (file, cb) => {
+  fs.stat(file, (err, stats) => {
+    if (err) {
+      cb(err);
+      return;
+    }
+    if (stats.isDirectory()) {
+      fs.readdir(file, (err, files) => {
+        if (err) {
+          cb(err);
+          return;
+        }
+        const rm = files.reduce(
```

Comment:
Can you also rewrite it to run multiple parallel asynchronous operations instead of sequential execution?